.prettierignore

```diff
@@ -1,2 +1,3 @@
 node_modules
 build
+public
```

Comment:
Is there a reason why we're committing the public folder now? PR seems to add a lot of noise files.

Reply:
That's where the trading view lib lives, but I can scope this to just trading view folder

Reply:
I see, I guess I just assumed TV was a plug and play npm package.

Reply:
I presume they keep it out of npm as it's kind of private, have to go through that process to get access